packages/react-components/react-data-grid-react-window/src/components/DataGridBody/DataGridBody.types.ts

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import type {
+  RowState,
+  DataGridBodyProps as DataGridBodyPropsBase,
+  DataGridBodySlots as DataGridBodySlotsBase,
+  DataGridBodyState as DataGridBodyStateBase,
+} from '@fluentui/react-components/unstable';
+
+export type DataGridBodySlots = DataGridBodySlotsBase;
+
+// Use any here since we can't know the user types
+// The user is responsible for narrowing the type downstream
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type RowRenderFunction = (row: RowState<any>, styles: React.CSSProperties) => React.ReactNode;
```

Comment:
```suggestion
export type RowRenderFunction = (row: RowState<any>, style: React.CSSProperties) => React.ReactNode;
```

As mentioned in another comments:
- may be use `unknown`
- Naming problem (type name does not include component name)
- nit: `style` instead of `styles` as it will be passed to `style` property

Reply:
naming is going to be tracked under https://github.com/microsoft/fluentui/issues/26198, as said before let's deal with the types offline together

Reply:
Updated `styles` to `style` in https://github.com/microsoft/fluentui/pull/26081/commits/db31ef681a2ee2caee4c988bc2d18683597ccc27